pkg/cinder/dbsync.go

```diff
@@ -64,6 +66,12 @@ func DbSyncJob(instance *cinderv1beta1.Cinder, labels map[string]string, annotat
 		},
 	}
 
+	// add CA cert if defined
```

Comment:
@Akrog same question as above; do we need the CA cert for DB sync?

Reply:
DB connection gets changed to TLS in follow up.

Reply:
It's the same case as the cronjob. Once DB connections uses TLS we will need it.